extensions/amp-accordion/1.0/accordion.js

```diff
@@ -37,6 +37,9 @@ import {useStyles} from './accordion.jss';
 const AccordionContext = Preact.createContext(
   /** @type {AccordionDef.ContextProps} */ ({})
 );
+const SectionContext = Preact.createContext(
+  /** @type {AccordionDef.ContextProps} */ ({})
```

Comment:
Should the type be different here?

Reply:
Thanks, yep, updated all types in the latest revision.